homeassistant/components/synology_dsm/switch.py

```diff
@@ -1,30 +1,42 @@
 """Support for Synology DSM switch."""
 from __future__ import annotations
 
+from dataclasses import dataclass
 import logging
 from typing import Any
 
 from synology_dsm.api.surveillance_station import SynoSurveillanceStation
 
-from homeassistant.components.switch import SwitchEntity
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity import DeviceInfo
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
 
 from . import SynoApi, SynologyDSMBaseEntity
-from .const import (
-    COORDINATOR_SWITCHES,
-    DOMAIN,
-    SURVEILLANCE_SWITCH,
-    SYNO_API,
-    SynologyDSMSwitchEntityDescription,
-)
+from .const import COORDINATOR_SWITCHES, DOMAIN, SYNO_API, SynologyDSMEntityDescription
```

Comment:
I think the `SynologyDSMEntityDescription` and ` SynologyDSMRequiredKeysMixin` should also be moved out of `const.py` it is weird to have `dataclass` there.

Reply:
will do this in a separate PR which will introduce a `model.py` for integration shared dataclasses